src/Chapter1.hs

```diff
@@ -478,7 +479,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = n `mod` 10
```

Comment:
One small note, it won't work correctly on the negative numbers, unfortunately.
Here you would need to use one more standard function – to take the __abs__olute value of n 🙂